GitHub.Collectors.Functions/GitHubFunctions.cs

```diff
@@ -652,6 +652,97 @@ public async Task Traffic([QueueTrigger("traffic")] string queueItem, ExecutionC
+            {
+                TypeNameHandling = TypeNameHandling.None
+            };
+            PointCollectorInput pointCollectorInput = JsonConvert.DeserializeObject<PointCollectorInput>(queueItem, serializerSettings);
+
+            FunctionContext context = new FunctionContext()
+            {
+                CollectorType = CollectorType.Point.ToString(),
+                CollectorIdentity = this.configManager.GetCollectorIdentity(),
+                FunctionStartDate = functionStartDate,
+                SessionId = sessionId,
+                InvocationId = executionContext.InvocationId.ToString(),
+                DequeueCount = dequeueCount,
+            };
+
+            ITelemetryClient telemetryClient = new GitHubApplicationInsightsTelemetryClient(this.telemetryClient, context, logger);
+            bool success = false;
+            string outputPaths = String.Empty;
+            Dictionary<string, string> sessionEndProperties = new Dictionary<string, string>();
+            StatsTracker statsTracker = null;
+            string identifier = "Point";
+            try
+            {
+                ICache<PointCollectorTableEntity> pointCache = new AzureTableCache<PointCollectorTableEntity>(telemetryClient, "point");
+                await pointCache.InitializeAsync().ConfigureAwait(false);
+                Repository repository = pointCollectorInput.getRepository();
+                ICache<RateLimitTableEntity> rateLimiterCache = new AzureTableCache<RateLimitTableEntity>(telemetryClient, "ratelimiter");
+                await rateLimiterCache.InitializeAsync().ConfigureAwait(false);
+                IRateLimiter rateLimiter = new GitHubRateLimiter(this.configManager.UsesGitHubAuth(context.CollectorType) ? repository.OrganizationLogin : "*", rateLimiterCache, this.httpClient, telemetryClient, maxUsageBeforeDelayStarts: 95.0, this.apiDomain, throwOnRateLimit: true);
+                ICache<ConditionalRequestTableEntity> requestsCache = new AzureTableCache<ConditionalRequestTableEntity>(telemetryClient, "requests");
+                await requestsCache.InitializeAsync().ConfigureAwait(false);
+                GitHubHttpClient httpClient = new GitHubHttpClient(this.httpClient, rateLimiter, requestsCache, telemetryClient);
+                IAuthentication authentication = this.configManager.GetAuthentication(CollectorType.Point, httpClient, repository.OrganizationLogin, this.apiDomain);
+
+                StorageManager storageManager;
+                List<IRecordWriter> recordWriters;
+                FunctionContextWriter<FunctionContext> contextWriter = new FunctionContextWriter<FunctionContext>();
+                using (storageManager = this.configManager.GetStorageManager(context.CollectorType, telemetryClient))
+                {
+                    recordWriters = storageManager.InitializeRecordWriters(identifier, context, contextWriter, this.adlsClient.AdlsClient);
```

Comment:
Hmmmm, at some point, let's revisit how we are using AdlsClient to initiate the record writers. I think this is the case in ADO collectors as well, but considering that we don't do directly ingestion to ADLS anymore (within the collectors), we should make this simpler. Can you leave a comment here so that we don't forget?